### PR TITLE
add localize for chinese-simple and chinese-traditional

### PR DESCRIPTION
**Pull request by hjw21century**
### Status
READY

### Description
Add localization for chinese-simple and chinese-traditional language.

### Steps to Test or Reproduce
open the Preference panel and click the Generral page, then change the pickable to "简体中文"  or "繁体中文"

### CODE REVIEW

**Comment by hjw21century** [comment]
image
image

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_bookmarks_new_bookmark_service" = "服务";
+"preferences_bookmarks_new_bookmark_already_exist" = "具有此链接/电话的书签已存在名称％@：\ n％@";
+"preferences_bookmarks_new_bookmark_error_title" = "无法添加书签";
+
+// MARK: - Preferences Calendars
+
+"preferences_calendars_select_calendars_title" = "选择日历以在状态栏中显示事件";
+"preferences_calendars_add_account_description" = "看不到您需要的日历？";
+"preferences_calendars_add_account_button" = "新增帐户";
+"preferences_calendars_add_account_modal" = "要添加外部日历，请执行以下步骤：\ n1。 打开默认日历应用程序\ n2。 单击菜单中的“添加帐户”\ n3。 选择并连接您的帐户";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:143]:
```suggestion
"preferences_calendars_add_account_modal" = "要添加外部日历，请执行以下步骤：\n1。 打开默认日历应用程序\n2。 单击菜单中的“添加帐户”\n3。 选择并连接您的帐户";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_general_option_preferred_language_english_value" = "英语";
+"preferences_general_option_preferred_language_russian_value" = "俄语";
+"preferences_general_option_preferred_language_chinese_simple_value" = "简体中文";
+"preferences_general_option_preferred_language_chinese_traditional_value" = "繁体中文";
+"preferences_general_option_shortcuts" = "快捷方式";
+"preferences_general_shortcut_open_menu" = "打开菜单：";
+"preferences_general_shortcut_create_meeting" = "创建会议：";
+"preferences_general_shortcut_join_next" = "加入下一次活动会议：";
+"preferences_general_shortcut_join_from_clipboard" = "加入剪贴板的会议：";
+"preferences_general_meeting_bar_description" = "会议栏是Andrii Leitsius创建的开源应用程序，Andrii Leitsius \ nthe app旨在使您的在线会议体验更顺畅，更容易";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:51]:
```suggestion
"preferences_general_meeting_bar_description" = "会议栏是Andrii Leitsius创建的开源应用程序，Andrii Leitsius \nthe app旨在使您的在线会议体验更顺畅，更容易";
```

Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:51]:
```suggestion
"preferences_general_meeting_bar_description" = "会议栏是Andrii Leitsius创建的开源应用程序，Andrii Leitsius\nthe app旨在使您的在线会议体验更顺畅，更容易";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:143]:
```suggestion
"preferences_calendars_add_account_modal" = "要添加外部日历，请执行以下步骤：\n1。 打开默认日历应用程序\n2。 单击菜单中的“添加帐户”\n3。 选择并连接您的帐户";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+// MARK: - Preferences Bookmarks
+
+"preferences_bookmarks_delete_bookmark_title" = "删除书签？";
+"preferences_bookmarks_delete_bookmark_message" = "你想删除书签％@吗？";
+"preferences_bookmarks_add_bookmark_button" = "添加书签";
+"preferences_bookmarks_new_bookmark_title" = "新的书签";
+"preferences_bookmarks_new_bookmark_name" = "name";
+"preferences_bookmarks_new_bookmark_link_phone" = "链接/电话";
+"preferences_bookmarks_new_bookmark_service" = "服务";
+"preferences_bookmarks_new_bookmark_already_exist" = "具有此链接/电话的书签已存在名称％@：\ n％@";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:135]:
```suggestion
"preferences_bookmarks_new_bookmark_already_exist" = "具有此链接/电话的书签已存在名称％@：\n％@";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+// MARK: - Preferences Services
+
+"preferences_services_link_meet_title" = "开放符合链接";
+"preferences_services_link_zoom_title" = "打开缩放链接";
+"preferences_services_link_team_title" = "开放团队链接";
+"preferences_services_link_default_browser_value" = "默认浏览器";
+"preferences_services_link_zoom_value" = "缩放应用程序";
+"preferences_services_link_teams_value" = "团队应用程序";
+"preferences_services_supported_links_list" = "支持的服务链接：\ n％@";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:119]:
```suggestion
"preferences_services_supported_links_list" = "支持的服务链接：\n％@";
```

File: MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_general_option_preferred_language_english_value" = "英語";
+"preferences_general_option_preferred_language_russian_value" = "俄語";
+"preferences_general_option_preferred_language_chinese_simple_value" = "簡體中文";
+"preferences_general_option_preferred_language_chinese_traditional_value" = "繁體中文";
+"preferences_general_option_shortcuts" = "快捷方式";
+"preferences_general_shortcut_open_menu" = "打開菜單：";
+"preferences_general_shortcut_create_meeting" = "創建會議：";
+"preferences_general_shortcut_join_next" = "加入下一次活動會議：";
+"preferences_general_shortcut_join_from_clipboard" = "加入剪跕板的會議：";
+"preferences_general_meeting_bar_description" = "會議欄是Andrii Leitsius創建的開源應用程序，Andrii Leitsius \ nthe app旨在使您的在線會議體驗更順暢，更容易";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings:51]:
```suggestion
"preferences_general_meeting_bar_description" = "會議欄是Andrii Leitsius創建的開源應用程序，Andrii Leitsius\nthe app旨在使您的在線會議體驗更順暢，更容易";
```

File: MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,337 @@
...
+
+// MARK: - Preferences Services
+
+"preferences_services_link_meet_title" = "開放符合鍊接";
+"preferences_services_link_zoom_title" = "打開縮放鍊接";
+"preferences_services_link_team_title" = "開放團隊鍊接";
+"preferences_services_link_default_browser_value" = "默認瀏攬器";
+"preferences_services_link_zoom_value" = "縮放應用程序";
+"preferences_services_link_teams_value" = "團隊應用程序";
+"preferences_services_supported_links_list" = "支持的服務鍊接：\ n％@";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings:119]:
```suggestion
"preferences_services_supported_links_list" = "支持的服務鍊接：\n％@";
```

File: MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,337 @@
...
+// MARK: - Preferences Bookmarks
+
+"preferences_bookmarks_delete_bookmark_title" = "刪除書籤？";
+"preferences_bookmarks_delete_bookmark_message" = "你想刪除書籤％@嗎？";
+"preferences_bookmarks_add_bookmark_button" = "添加書籤";
+"preferences_bookmarks_new_bookmark_title" = "新的書籤";
+"preferences_bookmarks_new_bookmark_name" = "name";
+"preferences_bookmarks_new_bookmark_link_phone" = "鍊接/電話";
+"preferences_bookmarks_new_bookmark_service" = "服務";
+"preferences_bookmarks_new_bookmark_already_exist" = "具有此鍊接/電話的書籤已存在名稱％@：\ n％@";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings:135]:
```suggestion
"preferences_bookmarks_new_bookmark_already_exist" = "具有此鍊接/電話的書籤已存在名稱％@：\n％@";
```

File: MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings [hunk 8]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_bookmarks_new_bookmark_error_title" = "無法添加書籤";
+
+// MARK: - Preferences Calendars
+
+"preferences_calendars_select_calendars_title" = "選擇日曆以在狀態欄中顯示事件";
+"preferences_calendars_add_account_description" = "看不到您需要的日曆？";
+"preferences_calendars_add_account_button" = "新增帳戶";
+"preferences_calendars_add_account_modal" = "要添加外部日曆，請執行以下步驟：\ n1。 打開默認日曆應用程序\ n2。 單擊菜單中的“添加帳戶”\ n3。 選擇並連接您的帳戶";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hant.lproj/Localizable.strings:143]:
```suggestion
"preferences_calendars_add_account_modal" = "要添加外部日曆，請執行以下步驟：\n1。 打開默認日曆應用程序\n2。 單擊菜單中的“添加帳戶”\n3。 選擇並連接您的帳戶";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 9]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_general_shortcut_create_meeting" = "创建会议：";
+"preferences_general_shortcut_join_next" = "加入下一次活动会议：";
+"preferences_general_shortcut_join_from_clipboard" = "加入剪贴板的会议：";
+"preferences_general_meeting_bar_description" = "会议栏是Andrii Leitsius创建的开源应用程序，Andrii Leitsius \ nthe app旨在使您的在线会议体验更顺畅，更容易";
+"preferences_general_external_patronage" = "惠顾";
+"preferences_general_external_gitHub" = "GitHub.";
+"preferences_general_external_contact" = "接触";
+"preferences_general_patron_title" = "成为赞助人";
+"preferences_general_patron_three_months" = "3个月 -  2.99美元";
+"preferences_general_patron_six_months" = "6个月 -  5.99 USD";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:57]:
```suggestion
"preferences_general_patron_six_months" = "6个月 -  5.99美元";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 10]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_general_option_preferred_language_chinese_simple_value" = "简体中文";
+"preferences_general_option_preferred_language_chinese_traditional_value" = "繁体中文";
+"preferences_general_option_shortcuts" = "快捷方式";
+"preferences_general_shortcut_open_menu" = "打开菜单：";
+"preferences_general_shortcut_create_meeting" = "创建会议：";
+"preferences_general_shortcut_join_next" = "加入下一次活动会议：";
+"preferences_general_shortcut_join_from_clipboard" = "加入剪贴板的会议：";
+"preferences_general_meeting_bar_description" = "会议栏是Andrii Leitsius创建的开源应用程序，Andrii Leitsius \ nthe app旨在使您的在线会议体验更顺畅，更容易";
+"preferences_general_external_patronage" = "惠顾";
+"preferences_general_external_gitHub" = "GitHub.";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:53]:
```suggestion
"preferences_general_external_gitHub" = "GitHub";
```

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 11]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_general_external_contact" = "接触";
+"preferences_general_patron_title" = "成为赞助人";
+"preferences_general_patron_three_months" = "3个月 -  2.99美元";
+"preferences_general_patron_six_months" = "6个月 -  5.99 USD";
+"preferences_general_patron_twelve_months" = "12个月 -  11.99美元";
+"preferences_general_patron_description" = "这些一次性购买不会自动更新。";
+"preferences_general_patron_thank_for_purchase" = "谢谢！ 您支持Meetabbar％DOMEN！ 🎉";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:60]:
```suggestion
"preferences_general_patron_thank_for_purchase" = "谢谢！ 您支持MeetingBar ％d OMEN！ 🎉";
```

Comment (leits) [reply MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:60]: OMEN?

File: MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings [hunk 12]
```diff
@@ -0,0 +1,337 @@
...
+"preferences_appearance_events_value_hide" = "隐藏";
+"preferences_appearance_events_value_as_inactive" = "显示为非活动";
+"preferences_appearance_events_value_as_underlined" = "表现为下划线";
+"preferences_appearance_events_value_with_strikethrough" = "用滴塞展示";
+"preferences_appearance_events_value_only_with_link" = "仅显示会议链接";
+"preferences_appearance_status_bar_title" = "状态栏";
+"preferences_appearance_status_bar_icon_title" = "图标";
+"preferences_appearance_status_bar_icon_app_icon_value" = "\ u00a0app图标";
+"preferences_appearance_status_bar_icon_calendar_icon_value" = "\ U00A0日历图标";
+"preferences_appearance_status_bar_icon_specific_icon_value" = "\ U00A0事件特定图标（I.N. MS团队）";
```
Comment (leits) [inline MeetingBar/Resources /Localization /zh-Hans.lproj/Localizable.strings:88]:
```suggestion
"preferences_appearance_status_bar_icon_specific_icon_value" = "\U00A0 事件特定图标（I.N. MS Teams）";
```

**Review by leits — CHANGES_REQUESTED** [review]
Hi, @hjw21century.
Thanks for the PR!
I review it and found a lot of typos.
Can you double-check localization?

**Comment by leits** [comment]
Closed because no response was received.

**Pull request closed without merging**